src/components/organism/sections/footer/index.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from "classnames";
+import { BsLinkedin, BsTwitter } from 'react-icons/bs';
+import { FooterList } from "../../../atoms/footer-list";
+import { footerLinks } from "@/model/api/routes-data/data";
+import NextLink from "next/link";
+import Image from "next/image";
+import FooterLinks from "@/components/molecules/footer-links";
+
+interface FooterProps {
+  className?: string;
+  kind?: "dark" | "light";
+}
+
+const social = [
+    { url: 'https://www.linkedin.com/company/getsalesassist', icon: <BsLinkedin />},
+    { url: 'https://twitter.com/SalesAssistSaaS', icon: <BsTwitter />}
+]
+
+export function Footer({ className, kind = "light" }: FooterProps) {
+  const baseClasses = classNames(
+    "md:py-12 duration-300 py-6 ",
+    { "bg-zinc-800": kind === "dark" },
+    { "text-white": kind === "dark" },
+    className
+  );
+
+  const socialClasses = classNames(
+    "text-[1.875rem] text-white rounded",
+    { "bg-white": kind === "dark" },
+    { "bg-zinc-800": kind === "light" }
+  );
+
+  return (
+    <footer className={baseClasses}>
+      <div className="h-full container mx-auto flex-grow flex gap-6 flex-col">
+        {/* footer left panel */}
+        <div className="h-1/2 w-full flex justify-between md:gap-4 md:items-center">
+          <div className="w-full md:w-fit lg:w-1/2 h-full px-2 md:py-2 relative mb-auto">
+            <div className="max-w-[164px]">
+              {kind == "dark" ? (
+                <Image
+                  src="/images/WLogo.svg"
+                  alt=""
+                  width={500}
+                  height={224}
+                />
+              ) : (
+                <Image
+                  src="/images/logo.png"
+                  alt={""}
+                  width={500}
+                  height={224}
+                />
+              )}
```

Comment:
**Change**: you can refactor this to a logo component that will render depending on the kind